homeassistant/components/logger/websocket_api.py

```diff
@@ -68,7 +68,7 @@ async def handle_integration_log_level(
             msg["id"], websocket_api.ERR_NOT_FOUND, "Integration not found"
         )
         return
-    await async_get_domain_config(hass).settings.async_update(
+    await hass.data[DATA_LOGGER].settings.async_update(
```

Comment:
It isn't great that we are accessing `hass.data` in tests, but its only in the integration tests now so thats still better.